src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -337,6 +385,14 @@ taken into account.
 Like most other powerful web APIs, the `getInstalledRelatedApps()` API is
 only available when served over **HTTPS**.
 
+## Still have questions? {: #questions }
+
+Still have questions? Check [`getInstalledRelatedApps` on StackOverflow][so-gira]
```

Comment:
```suggestion
Still have questions? Check [`getInstalledRelatedApps()` on StackOverflow][so-gira]
```

Reply:
Made it more explict - "check the `getInstalledRelatedApps` tag", the tag doesn't include the brackets.